packages/editor/src/panels/hierarchy/hierarchynode.tsx

```diff
@@ -430,10 +430,10 @@ export default React.memo(function HierarchyTreeNode(props: ListChildComponentPr
         />
         <div
           className={twMerge(
-            'flex w-full items-center justify-between gap-x-2 bg-inherit pr-2',
-            rootEntity === entity ? 'p-2' : 'py-1 pl-10 pr-2'
+            'flex items-center justify-between gap-x-2 bg-inherit pr-2',
+            rootEntity === entity ? 'p-2' : 'py-1 pr-2'
           )}
-          style={{ marginLeft: `${node.depth * 0.75}rem` }}
+          style={{ width: `calc(100% - ${node.depth * 0.75}rem)`, marginLeft: `${node.depth * 0.75}rem` }}
```

Comment:
🐑 We shouldnt have to use `calc()` and JS to align something to the right edge, even with cascading children. I'm sure this works well and it will be fine for now, but its a pretty brittle solution for something that CSS can handle smoothly without JS or assumptions, like the `0.75`, which will prob change in the future, on certain screen sizes, or when someone changes/adds another element

Reply:
I just realized I can remove this. I'll update my PR and merge it.